src/pages/cartPage/cartPageUtils.js

```diff
@@ -0,0 +1,217 @@
+import axios from "axios";
+const incrementItemInCartAndSetCart = ({ itemDetails, cart, setCart }) => {
+  // setCart((cart) =>
+  //   cart.map((e) => {
+  //     e["id"] === itemDetails["id"] ? { ...e, qty: e.qty + 1 } : e;
+  //   })
+  // );
```

Comment:
remove commented codes

Reply:
This is a feature, it's not working as of now will be asking the same in DSH.